producers/src/main/java/uk/gov/moj/sdt/producers/sdtws/SdtEndpointPortType.java

```diff
@@ -267,4 +267,9 @@ public void setWsReadSubmitQueryHandler(final IWsReadSubmitQueryHandler wsReadSu
         this.wsReadSubmitQueryHandler = wsReadSubmitQueryHandler;
     }
 
+    public String createClaimsDefenceJSON() {
```

Comment:
seems this is not used, please delete this

Reply:
Done